packages/core/src/render/api.ts

```diff
@@ -241,3 +241,15 @@ export function injectRenderer2(): Renderer2 {
   const nodeAtIndex = getComponentLViewByIndex(tNode.index, lView);
   return (isLView(nodeAtIndex) ? nodeAtIndex : lView)[RENDERER] as Renderer2;
 }
+
+/**
+ * This enum is meant to be used by `ɵtype` properties of the different renderers implemented
+ * by the framework
+ *
+ * We choose to not add `ɵtype` to `Renderer2` to no expose it to the public API.
+ */
+export const enum RendererTypes {
+  Default,
+  Animation,
+  AnimationDelegated,
+}
```

Comment:
I was thinking about this a bit more and I think we should limit the type to only animation renderers. The thing is that custom renderer factories can be provided via public APIs, so we can no rely on the `Default` at all (in case of a custom renderer, the `type` field would not exist).

The proposal is to do something like this instead:
```suggestion
export const enum AnimationRendererType {
  Regular,
  Delegated,
}
```